src/components/input.ts

```diff
@@ -1,6 +1,7 @@
 import { TooltipTrigger, useWindowDimensions } from '@terra-ui-packages/components';
 import Downshift from 'downshift';
 import _ from 'lodash/fp';
+import React from 'react';
```

Comment:
input.ts was part of a change I decided to back out of,. But here I kept the effort I put into a partial Typescript conversion.  No functional changes... some minor Typesecript-surfaced linting errors fixed.